backend/app/models/user.rb

```diff
@@ -40,11 +42,11 @@ def drafts
   end
 
   def admin
-    role == 'admin'
+    admin?
   end
 
   def staff
-    role == 'staff'
+    author?
```

Comment:
This is wrong. Can you for now just send `editor?` instead?  I don't really think we're using `staff` in `web-client` but just in case for now that should work.